tests/functional/lib/helpers.js

```diff
@@ -336,8 +336,9 @@ define([
     startListening();
   }
 
-  function openVerificationLinkDifferentBrowser(client, email) {
+  function openVerificationLinkDifferentBrowser(client, email, emailNumber) {
     if (typeof client === 'string') {
+      emailNumber = email;
```

Comment:
`client` is optional, a remnant from when it was required. So here I am shifting each argument right.